packages/itwinui-css/src/field.scss

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+:where(.iui-field) {
+  --_iui-field-color-text: var(--iui-color-text);
+  --_iui-field-color-text-hover: var(--iui-color-text-hover);
+  --_iui-field-color-text-disabled: var(--iui-color-text-disabled);
+  --_iui-field-color-background: var(--iui-color-background);
+  --_iui-field-color-background-hover: var(--iui-color-background-hover);
+  --_iui-field-color-background-disabled: var(--iui-color-background-disabled);
+  --_iui-field-color-border: var(--iui-color-border-foreground);
+  --_iui-field-color-border-hover: var(--iui-color-border-foreground-hover);
+  --_iui-field-color-border-disabled: var(--iui-color-border-disabled);
+  --_iui-field-color-focus-outline: var(--iui-color-border-accent);
+  --_iui-field-height: var(--iui-component-height);
+  --_iui-field-focus-outline-offset: -2px;
+  --_iui-field-focus-outline-width: 2px;
+  --_iui-field-font-size: var(--iui-font-size-1);
+
+  background-color: var(--_iui-field-color-background);
+  border: 1px solid var(--_iui-field-color-border);
+  border-radius: var(--iui-border-radius-1);
+  color: var(--_iui-field-color-text);
+  font-family: inherit;
+  font-size: var(--_iui-field-font-size);
+  font-weight: var(--iui-font-weight-normal);
+  min-block-size: var(--_iui-field-height);
+  min-inline-size: var(--_iui-field-height);
+  -webkit-tap-highlight-color: var(--_iui-field-color-background-hover);
+  transition:
+    background-color var(--iui-duration-1) ease-out,
+    border-color var(--iui-duration-1) ease-out,
+    color var(--iui-duration-1) ease-out;
+
+  &:where(:hover) {
+    background-color: var(--_iui-field-color-background-hover);
+    border-color: var(--_iui-field-color-border-hover);
+    color: var(--_iui-field-color-text-hover);
+  }
+
+  &:where(:active) {
```

Comment:
`:hover` and `:active` can be combined?

Reply:
Jeeeeez, how did I miss this one. 🤦🏻‍♂️ Cleaned up.

Reply:
You can combine it even further using `:where(:hover, :active)`

Reply:
Simplified even further.